app/page.tsx

```diff
@@ -37,17 +40,25 @@ export default function Home() {
         <div className="flex gap-4 md:gap-6 justify-center">
           <div className="text-center mt-6">
             <Button
-              onClick={() => {setLoading(true); router.push('/room')}}
+              onClick={() => {
+                setLoading(true);
+                router.push("/room");
+              }}
               className="tracking-wider text-xs md:text-sm"
+              id="room-btn"
```

Comment:
Remove the `ids` that are added in tsx file for testing.

You can test without adding ids. Please see my comments in the test files